clients/cobol-lsp-vscode-extension/src/services/copybook/downloader/DownloadUtil.ts

```diff
@@ -153,9 +153,13 @@ export class DownloadUtil {
     documentUri: string,
     copybookNames: CopybookName[],
   ): boolean {
-    const dialects = new Set(
-      copybookNames.map((n) => n.dialect?.toLocaleUpperCase()).filter(Boolean),
-    );
+    const dialects = [
+      ...new Set(
+        copybookNames
+          .map((n) => n.dialect?.toLocaleUpperCase())
+          .filter(Boolean),
+      ),
+    ];
```

Comment:
This looks like an unnecessary copy, the original code looks ok (better even).

Reply:
This removes duplicates

Reply:
so did the original code.